Update compression config for openlm-research/open_llama_3b_v2

nikita-savelyevv opened the pull request:
Reduce PPL for compressed `openlm-research/open_llama_3b_v2` by removing `all_layers=True` and adding AWQ.

| Precision | all_layers | AWQ | PPL   |
|-----------|------------|-|-------|
| FP16      |            | | 12.40 |
| INT4_SYM | False     | False | 13.45 |
| INT4_SYM | True      | False | 13.73 |
| INT4_ASYM | False   | True  | 12.77 |
| INT4_ASYM | False    | False | 13.15 |
| INT4_ASYM | True     | False |  13.36 |




## Before submitting
- [ ] This PR fixes a typo or improves the docs (you can dismiss the other checks if that's the case).
- [ ] Did you make sure to update the documentation with your changes?
- [ ] Did you write any new necessary tests?

nikita-savelyevv commented:
@KodiaqQ please take a look

KodiaqQ commented:
@nikita-savelyevv, you've provided numbers in the description with the ASYM mode, while configuration contains `"sym": True`. Why so?

nikita-savelyevv commented:
@KodiaqQ Thanks for noticing. There were some errors in the collected metrics. I've updated the metrics in the PR description and changed the mode to asymmetric.

HuggingFaceDocBuilderDev commented:
The docs for this PR live [here](https://moon-ci-docs.huggingface.co/docs/optimum-intel/pr_860). All of your documentation changes will be reflected on that endpoint. The docs are available until 30 days after the last update.
